lib/itoolkit.js

```diff
@@ -569,7 +569,29 @@ class iSql {
   }
 }
 
-
+exports.types = {
+    char(countChars) {
+        return countChars.toString() + "A";
+    },
+    packed(digits, delimiter) {
+        return digits.toString() + "P" + delimiter.toString();
+    },
+    zoned(digits, delimiter) {
+        return digits.toString() + "S" + delimiter.toString();
+    },

```

Comment:
I'd suggest changing `digits` and `delimiter` to something more descriptive. SQL uses the terms `precision` and `scale`, though some may find that confusing. Perhaps `total_digits` and `decimal_digits` would be more appropriate?